### PR TITLE
Fix submitting when allowed attempts is 1

### PR DESCRIPTION
**Pull request by armstrongnate**
refs: MBL-14607
affects: student
release note: Fixed submitting to assignments that only allow one attempt

Test plan: see ticket

### CODE REVIEW

File: Core/Core/Submissions/Submission.swift [hunk 1]
```diff
@@ -132,7 +132,7 @@ extension Submission: WriteableModel {
         model.workflowState = item.workflow_state
         model.latePolicyStatus = item.late_policy_status
         model.pointsDeducted = item.points_deducted
-        model.attempt = item.attempt ?? 1
+        model.attempt = item.attempt ?? 0
```
Comment (armstrongnate) [inline Core/Core/Submissions/Submission.swift:135]: @vanwagonet is there a reason we defaulted to 1 and not 0?

Comment (vanwagonet) [reply Core/Core/Submissions/Submission.swift:135]: lol, no.

**Comment by inst-danger** [comment]
## Latest Installable Builds

(will redirect to this page if no builds exist yet)
student: [Link](https://rebrand.ly/swfh5uc) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fswfh5uc)
teacher: [Link](https://rebrand.ly/4rjcpus) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2F4rjcpus)
parent: [Link](https://rebrand.ly/ie1atiz) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fie1atiz)

**Comment by inst-danger** [comment]
Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 93.35% | 93.35% | 0%

[MBL-14607](https://instructure.atlassian.net/browse/MBL-14607)

#### Affected Apps: Student

#### Release Note: 
Fixed submitting to assignments that only allow one attempt
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Pull request merged**